guides/micronaut-creating-first-graal-app/micronaut-creating-first-graal-app.adoc

```diff
@@ -98,25 +98,32 @@ Start the application either using Docker or the native executable. You can run
 
 [source,bash]
 ----
-complete $ time curl localhost:8080/conferences/random
+time curl localhost:8080/conferences/random
+----
+
+```
```

Comment:
I am not sure that this is property asciidoc. I think they are Github fenced code blocks